### PR TITLE
Releasing 2.15.0

### PR DESCRIPTION
**Pull request by jadahiya-MSFT**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

> Summarize the changes, including the goals and reasons for this change. Be sure to call out any technical or behavior changes that reviewers should be aware of.

> If this Pull Request should close/resolve any issues when merged, use [the special syntax for that](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) here.

### Main changes in the PR:

1.  Updating changelog and hash values for 2.15.0 release

### CODE REVIEW

**Review by KangxuanYe — DISMISSED** [review]

File: packages/teams-js/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+## 2.15.0
+
+Wed, 06 Sep 2023 20:30:49 GMT
+
+### Minor changes
+
+- Removed `appNotification` capability and collateral
+- Added `clipboard` capability, allowing access to the system clipboard programmatically
+- Fixed bugs within the `appNotification` capability and its validation
+- Add MacOS support
```
Comment (lakhveerk) [inline packages/teams-js/CHANGELOG.md:16]: Add**ed** MacOS support

Comment (jadahiya-MSFT) [reply packages/teams-js/CHANGELOG.md:16]: good catch, seems this passed through. I'll manually update the file

**Review by lakhveerk — APPROVED** [review]
Left a small comment. Otherwise lgtm

**Review by lakhveerk — APPROVED** [review]

File: packages/teams-js/CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
 
+## 2.15.0
+
+Wed, 06 Sep 2023 20:30:49 GMT
+
+### Minor changes
+
+- Removed `appNotification` capability and collateral
+- Added `clipboard` capability, allowing access to the system clipboard programmatically
+- Fixed bugs within the `appNotification` capability and its validation
```
Comment (AE-MS) [inline packages/teams-js/CHANGELOG.md:15]: In light of the line two before this, maybe you could just take this out to avoid confusion.

**Review by AE-MS — CHANGES_REQUESTED** [review]
:clock1:

File: packages/teams-js/CHANGELOG.md (hunk 2 above)
Comment (jadahiya-MSFT) [reply packages/teams-js/CHANGELOG.md]: Removed, thanks for pointing this out.

**Review by AE-MS — APPROVED** [review]
:shipit:

**Pull request merged**